Libraries/PushNotificationIOS/PushNotificationIOS.js

```diff
@@ -485,6 +487,13 @@ class PushNotificationIOS {
   }
 
   /**
+   * Gets the content-available number from the `aps` object
+   */
+  getContentAvailable(): ?number {
```

Comment:
Can we declare special type `type ContentAvailable = 1;` (https://flow.org/en/docs/types/unions/) and then use it as `?ContentAvailable`? (I am not sure.) Or maybe we can create special already optional type?